setup.py

```diff
@@ -85,7 +85,7 @@ def run(self):
 
 setup(
     name='sphinx_rtd_theme',
-    version='0.4.3.dev0',
+    version='0.4.5.dev0',
```

Comment:
This seems like a typo (also in `__init__.py`)

Reply:
Yeah, I've done this weirdly but I think it's correct. This is set to `0.4.4` (here and in `__init__.py`) in [`HEAD~`](https://github.com/readthedocs/sphinx_rtd_theme/pull/900/commits/a701997e75c8b53868584d0807dc61541db3451c) and changed to `0.4.5.dev0` in [`HEAD`](https://github.com/readthedocs/sphinx_rtd_theme/pull/900/commits/f8cc1843c504b7555d2bcade19af3ecf25f88780)

Reply:
Let me know if you want me to pull out the post-version bump patch. If not, you just need to make sure you take `HEAD~`, *not* `HEAD` :smile:

Reply:
Ah, now I understand what you did. This should also work, provided there are no additional commits needed to make 0.4.4 work.

Since there may be someone else working on this, I think it might be safer to keep this on `0.4.4`, go through all release shennanigans, then ciommit a version bump. How does that sound?

Reply:
Yup, good call (y) Thinking about it, this wouldn't work if you ended up with a merge commit either. Will drop that shortly